### ISSUE TITLE
nerofest / skill command not running on turn 3, proceed to auto battle without using skills

### ISSUE THREAD

**Issue by blashmer** [29988122/Fate-Grand-Order_Lua#240]
Hello,
I'm unable to make the script use my commands on turn 3

Android 9 running with ankulua 8.3.0 trial
English server

Here is my config file

[FGO_EN_REGULAR_test.lua.txt](https://github.com/29988122/Fate-Grand-Order_Lua/files/3569078/FGO_EN_REGULAR_test.lua.txt)

my Skill_Command = "c4,#,abg2hix31g4,#,efhi2j5"
It does everything fine but the last part after the second "#"
The script keeps running and proceed to do normal attacks without using skills.

If i fail to do the second battle in one turn, the script proceed to do the 3rd part of my command skill. It's like the "#" is registered for turn and not for battle.
The last battle is a "fatal battle", they show the enemy team a bit longer than in a normal battle.
I don't know if it's related

Do you have any idea? I feel like i'm doing a silly mistake

Thanks for your time!

**Comment by TryBane** [comment]
Update your Ankulua version.

**Comment by blashmer** [comment]
Thanks for your fast reply.

I did update to 8.5.2 trial but unfortunately the behavior is the same

**Comment by blashmer** [comment]
I also did another test on doors (QP) to check if my theory of "fatal battle" was true.
results, my guess was wrong, it's not working either, i see a click on left top corner and bottom right corner then it click on attack and play without using skills.

**Comment by aqer** [comment]
How about 8.5.0? I had met some strange problem on 8.5.2 before.

**Comment by TryBane** [comment]
Welp, the only thing I can think of is that the script is not recognizing the change in the wave. I assumed this was a result of the ankulua version not being up to date. I am unsure what the issue may be, so my next advice would be to redownload the script and try it again.

**Comment by blashmer** [comment]
> How about 8.5.0? I had met some strange problem on 8.5.2 before.
> So I go back to use 8.5.0

That's it ! It works ! wow that's super weird.
Thanks for your help guys <3<3<3

**Comment by potchy** [comment]
Open `/modules/battle.lua` with a text editor, insert a new line just above 127.
https://github.com/29988122/Fate-Grand-Order_Lua/blob/c641dba72dd95df798b9c24e5067b35dde9cf655/modules/battle.lua#L127

The result will be this:
```lua
_game.BATTLE_STAGE_COUNT_REGION:highlight()
return not _game.BATTLE_STAGE_COUNT_REGION:exists(currentStagePattern)
```

Now run the script again, enter a quest and when a red border appears next to the stage counter, take a screenshot and attach it here.

**Comment by TryBane** [comment]
Haha, debug unneeded now potchy, but thanks :D

**Comment by blashmer** [comment]
Thanks potchy !

**Comment by blashmer** [comment]
solved by using 8.5.0
